### PR TITLE
Add isReady methods

### LINKED ISSUE #1333: [Enhancement] isReady method
Hey !

Thanks for wavesurfer it's awesome.
I was thinking about adding an isReady() function wich simply return true or false. The event 'ready' is working and I can recreate the isReady function by myself I know but that implies I can not unsub the event from another callback.

If you have any idea. Thanks ! :)

### PR DESCRIPTION
**Pull request by Guisch**
### Short description of changes:
Add an isReady method to know if the wavesurfer is loaded and ready or not.

### Todos/Notes:
Need to be tested

### Related Issues and other PRs:
fixes #1333

### CODE REVIEW

File: src/wavesurfer.js [hunk 1]
```diff
@@ -894,6 +894,16 @@ export default class WaveSurfer extends util.Observer {
...
     }
 
+    /**
+     * Get the current ready status.
+     *
+     * @example const isReady = wavesurfer.isReady();
+     * @return {boolean}
+     */
+    isReady() {
+        return this.isMuted;
```
Comment (thijstriemstra) [inline src/wavesurfer.js]: @Guisch what does `isMuted` have to do with this? :)

Comment (Guisch) [reply src/wavesurfer.js]: Ahah, thanks Copy/Paste

**Comment by thijstriemstra** [comment]
@Guisch if there's a `isReady` attribute, why add a method that returns this property, why not use this property directly?

**Comment by Guisch** [comment]
Because it's a private attribute ? (it's the same logic for isMuted attribute and getMute function).
https://github.com/katspaugh/wavesurfer.js/blob/07306849a71db2f02c607bd53f8d06003b315c8a/src/wavesurfer.js#L356

**Pull request merged**